packages/tangle/tipmanager.go

```diff
@@ -375,7 +375,7 @@ func (t *TipManager) isPastConeTimestampCorrect(messageID MessageID) (timestampV
 	}
 
 	// if last confirmed message if older than minSupportedTimestamp, then all tips are invalid
```

Comment:
Can you change the comment here? This condition should be now very rare.

Reply:
This check has been actually removed in @piotrm50 PR.

Reply:
Done